File: Kakao/2018 1st Blind Recruitment/1.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(int n, vector<int> arr1, vector<int> arr2) {
    vector<string> answer(n);
    
    for(int i=0; i<n; ++i){
        int sum = arr1[i] | arr2[i];
        for(int j=n; sum > 0 || j > 0; --j){
            answer[i] = (sum % 2 ? "#" : " ") + answer[i];
            sum /= 2;
        }
    }
    
    return answer;
}

File: Kakao/Blind Recruitment/2019/1.py
# Problem 1

def solution(record):
    name_of = dict()
    
    for r in record:
        s = r.split()
        if s[0] != "Leave":
            name_of[s[1]] = s[2]
            
    answer = []
    for r in record:
        s = r.split()
        if s[0] != "Change":
            answer.append("{}님이 {}습니다.".format(name_of[s[1]], "들어왔" if s[0] == "Enter" else "나갔"))
    
    return answer